tdrs-backend/tdpservice/parsers/row_schema.py

```diff
@@ -114,7 +114,9 @@ def run_field_validators(self, instance, generate_error):
             else:
                 value = getattr(instance, field.name, None)
 
-            if field.required and not value_is_empty(value, field.endIndex-field.startIndex):
+            is_empty = value_is_empty(value, field.endIndex-field.startIndex)
```

Comment:
Ya I didn't think we wanted to add zero fill to the `value_is_empty` function since there are so many fields across the schemas that expect zero as a valid value. I will point Alex at this as well to make sure everything is good.

Reply:
@ADPennington can you verify that this is the logic we want in these situations?

Reply:
not sure i understand how we are using "#" and "@" but im seeing the expected behavior @elipe17 @jtimpe